nomos-da/kzgrs/benches/rs.rs

```diff
@@ -22,10 +25,10 @@ fn rs_encode(bencher: Bencher, size: usize) {
             let (_, poly) = bytes_to_polynomial::<31>(buff, domain).unwrap();
             let domain = GeneralEvaluationDomain::<Fr>::new(size * 2).unwrap();
             black_box(move || encode(&poly, domain))
-        })
+        });
 }
 
-#[divan::bench(args = [16399, 32798, 65565, 131099, 262167, 524241, 1048606], sample_size = 10, sample_count = 100)]
+#[divan::bench(args = [16399, 32798, 65565, 131_099, 262_167, 524_241, 1_048_606], sample_size = 10, sample_count = 100)]
```

Comment:
Let's fill the other as well maybe? `16_399`, `32_798`..?

Reply:
Clippy did not complain about those, but sure I can do that.

Reply:
Did that in https://github.com/logos-co/nomos-node/pull/1093/commits/a95373539ea37b68c60b52d0b366c17f8f5a453e. Although because there's no Clippy lint, 4 and 5 digit numbers won't be enforced to use this rule in the future.